### PR TITLE
Update data form

### PR DESCRIPTION
**Pull request by apanjain**

### CODE REVIEW

**Comment by apanjain** [comment]
@rohitjha941 new field is added for placement, please update the api call for create mentor

File: backend/pages/admin.py [hunk 1]
```diff
@@ -77,6 +77,11 @@ class MentorIntern(ImportExportModelAdmin):
     list_display = ('mentor', 'company', 'duration', 'domain', )
 
 
+@register(MentorPlacement)
+class MentorIntern(ImportExportModelAdmin):
```
Comment (rohitjha941) [inline backend/pages/admin.py:81]: MentorIntern ?

File: frontend/src/routes/DataCollection/MentorForm.js [hunk 2]
```diff
@@ -538,6 +548,50 @@ class MentorForm extends Component {
                 Add Internship
               </button>
             </div>
+            <br />
```
Comment (hacksdump) [inline frontend/src/routes/DataCollection/MentorForm.js:551]: Line breaks should not be added in HTML. Set the required gap with CSS.

File: frontend/src/routes/DataCollection/MentorForm.js [hunk 3]
```diff
@@ -538,6 +548,50 @@ class MentorForm extends Component {
...
+            {/* Placement */}
+
+            <div className={styles["form-group"]}>
+              <label htmlFor="placement">Placement (for final year only)</label>
+              <div className={`${styles["form-group"]} ${styles.row}`}>
+                <label
+                  htmlFor={"company"}
+                  className={`${styles["col-sm-2"]} ${styles["col-form-label"]}`}
+                >
+                  Name of Commpany:
```
Comment (hacksdump) [inline frontend/src/routes/DataCollection/MentorForm.js:563]:
```suggestion
                  Name of Company:
```

**Pull request merged**